pkgs/development/python-modules/bencoder/default.nix

```
@@ -0,0 +1,22 @@
+{ lib
+, python3Packages
+, fetchPypi
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "bencoder";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-rENvM/3X51stkFdJHSq+77VjHvsTyBNAPbCtsRq1L8I=";
+  };
+
+  buildInputs = with python3Packages; [ setuptools ];
```

```suggestion
  buildInputs = [ setuptools ];
```
Please add setuptools to inputs. Also this is likely nativeBuildInputs